docs/src/pages/guides/important-defaults.md

```diff
@@ -15,6 +15,8 @@ Out of the box, React Query is configured with **aggressive but sane** defaults.
   - The network is reconnected.
   - The query is optionally configured with a refetch interval.
 
+If you see a refetch that you are not expecting, it is likely because you just focused the window and React Query is doing a `refetchOnWindowFocus`. During development, this will probably be triggered more frequently, especially because focusing between the Browser DevTools and your app will also cause a fetch, so be aware of that.
```

Comment:
Can you move it down one paragraph, as the next paragraph references the list above and this kinda breaks it apart, thanks

Reply:
Forget that, it looks fine 😅